lutris/runners/ryujinx.py

```diff
@@ -0,0 +1,95 @@
+# Standard Library
+import filecmp
+import os
+from gettext import gettext as _
+from shutil import copyfile
+
+# Lutris Modules
+from lutris.runners.runner import Runner
+from lutris.util import system
+from lutris.util.log import logger
+
+
+
+class ryujinx(Runner):
+    human_name = _("Ryujinx")
+    platforms = [_("Nintendo Switch")]
+    description = _("Nintendo Switch emulator for LDN build visit https://ryujinx.org/")
+    runnable_alone = True
+    runner_executable = "ryujinx/publish/Ryujinx"
+    download_url = "https://ci.appveyor.com/api/buildjobs/o37yny5mfr4dyx6d/artifacts/ryujinx-1.0.7032-linux_x64.tar.gz"
```

Comment:
This is the only thing that stands out to me. Should the download URL really be hard-coded?